omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,96 @@
+#. Add all the authors of the paper to this new public group.
+
+Data setup
+----------
+
+#. Move the original images into the dedicated group using OMERO.web or
+   :doc:`OMERO.cli </users/cli/chgrp>`. The CLI is best used where Images or
+   Datasets are cross-linked to other Datasets or Projects in the original
+   group. The command ``bin/omero chgrp Project:$ID --include Dataset,Image``
+   cuts the cross-links in the original group and preserves the
+   Project/Dataset/Image hierarchy prepared for the move by the author.
+#. If you have used OMERO.figure to create your figures for publication, you
+   can always find the original data by using the 'info' tab, as shown in the
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
+#. Having all the data belong to one user simplifies the UI experience for
+   public users. If necessary, ownership of data can be transferred using the
+   'Chown' privilege (see :doc:`restricted-admins` and
+   :doc:`/users/cli/chown`).
+
+Configuring OMERO.web
+---------------------
+
+#. Configure the OMERO.web filter on the public user by setting the public URL
+   filter to allow 'webclient' (see :doc:`public` and
+   :property:`omero.web.public.url_filter`) so that the full webclient is
+   visible for the public user, and thus also the Data tree with Projects and
+   Datasets is browsable as well as Tags tab and full viewer of the images
+#. In order to have both a public webclient and a closed webclient side by
+   side without affecting the existing closed webclient (and thus giving your
+   existing non-public OMERO users an unchanged user experience in their
+   workflows), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at an existing
+   OMERO.server.
+
+Data configuration
+------------------
+
+#. Once the data is in the dedicated public group, it can be reorganized and
+   renamed to reflect the publication e.g. Projects can be renamed according
+   to the corresponding figure panels in the manuscript while the names of the
+   Datasets could be retained corresponding to different treatment conditions
+   represented in each figure panel. For example, Project
+   `Schleicher_etal_figure7_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27920>`_
+   contains images underlying the publication Figure panel 7c. Some Projects
+   underlie two publication Figure panels, such as Project
+   `Schleicher_etal_figure2_a_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27917>`_
+   where representative images are shown in the first panel and the
+   corresponding quantification is shown in the next panel. This makes clear
```

Comment:
I think we should link to the panels a, c of figure 2 in the publication itself here, and then use wording ``panel a``, ``panel c``, instead of ``first pnael....next panel``